var/spack/repos/builtin/packages/libcircle/package.py

```diff
@@ -5,14 +5,23 @@
 
 from spack import *
 
-
 class Libcircle(AutotoolsPackage):
     """libcircle provides an efficient distributed queue on a cluster,
        using self-stabilizing work stealing."""
 
     homepage = "https://github.com/hpc/libcircle"
+    git      = "https://github.com/hpc/libcircle.git"
+    url      = "https://github.com/hpc/libcircle/releases/download/0.2.1-rc.1/libcircle-0.2.1-rc.1.tar.gz'url='https://github.com/hpc/libcircle/releases/download/0.2.1-rc.1/libcircle-0.2.1-rc.1.tar.gz"
 
-    version('0.2.1-rc.1', sha256='5747f91cf4417023304dcc92fd07e3617ac712ca1eeb698880979bbca3f54865',
-            url='https://github.com/hpc/libcircle/releases/download/0.2.1-rc.1/libcircle-0.2.1-rc.1.tar.gz')
+    version('master', branch='master')
+    version('develop', branch='master')
```

Comment:
Did you mean to have a single master version, a single develop version, or both?